WPMedia/fusion-news-theme-blocks
# THEMES-1580 added transform to related-content

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

There appears to be an inconsistency in results returned by the related-content content source between Themes 1.X and Themes 2.0.X. In T1, the node with returned content is content_elements, in T2 it's basic . I assume this makes blocks that are using this CS (like OOTB top-table-list) display no results. 

## Jira Ticket

- [THEMES-1580](https://arcpublishing.atlassian.net/browse/THEMES-1580)

## Acceptance Criteria

The related-content-content-source-block should return an object with a `content_elements` field instead of a `basic` field

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-1580`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/related-content-content-source-block`
3. Visit this debugger page: http://localhost/pagebuilder/tools/debugger?_website=daily-intelligencer&type=content-debugger&source=related-content&config={%22_id%22:%22RYXYKZSMHVGAVFLSQIMS4PHWXM%22}
4. Verify that the content source returns an object with a `content_elements` array instead of a `basic` array
5. Add a list block to a `daily-intelligencer` test page in Pagebuilder, and set the content source to `related-content`
6. Set the ID for the content source to `RYXYKZSMHVGAVFLSQIMS4PHWXM`
7. Verify that the list successfully renders the content_elements

## Effect Of Changes

### Before

_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After

_Example: When I clicked the search button, the button turned green._

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1580]: https://arcpublishing.atlassian.net/browse/THEMES-1580?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
